process/slash/notifier/slashingNotifier_test.go

```diff
@@ -266,10 +267,15 @@ func TestSlashingNotifier_CreateShardSlashingTransaction_MultipleProposalProof(t
 			}, nil
 		},
 	}
-
-	expectedData := []byte(fmt.Sprintf("%s@%s@%d@%d@%s@%s", notifier.BuiltInFunctionSlashCommitmentProof,
-		[]byte{slash.MultipleProposalProofID}, shardID, round, []byte{byte('c'), byte('d')}, []byte("signature")))
-
+	shardBigInt := big.NewInt(int64(shardID))
+	roundBigInt := big.NewInt(int64(round))
+	expectedData := []byte(fmt.Sprintf("%s@%s@%s@%s@%s@%s",
```

Comment:
can you add the expected data as a const string here?

Reply:
Yes, changed it